permissions/src/androidLibAndroidTest/kotlin/BluetoothPermissionManagerTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.splendo.mpp.permissions
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.newSingleThreadContext
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+class BluetoothPermissionManagerTest {
+    private val mainThreadSurrogate = newSingleThreadContext("UI thread")
```

Comment:
There is a base test class that already does this.

Reply:
Though I'm surprised this is need in *androidTest* rather than just test? Maybe this was moved?